pkg/filter/stream/jwtauthn/filter.go

```diff
@@ -0,0 +1,59 @@
+package jwtauthn
+
+import (
+	"context"
+	"mosn.io/api"
+	"mosn.io/mosn/pkg/log"
+	"mosn.io/mosn/pkg/types"
+	"mosn.io/mosn/pkg/variable"
+	"mosn.io/pkg/buffer"
+)
+
+type filter struct {
+	config  FilterConfig
+	handler api.StreamReceiverFilterHandler
+}
+
+func newJwtAuthnFilter(config FilterConfig) *filter {
+	return &filter{
+		config: config,
+	}
+}
+
+func (f *filter) OnReceive(ctx context.Context, headers api.HeaderMap, buf buffer.IoBuffer, trailers api.HeaderMap) api.StreamFilterStatus {
+	log.DefaultLogger.Debugf("call jwt_authn filter")
+
+	// TODO(huangrh): bypass
+
+	requestArg, err := variable.GetVariableValue(ctx, types.VarHttpRequestArg)
+	if err != nil {
+		log.DefaultLogger.Errorf("[jwt_authn filter] get query parameter: %v", err)
+		return api.StreamFilterStop
```

Comment:
Not all requests have request parameters, so i think `api.StreamFilterContinue` should be returned

Reply:
ok